### ISSUE TITLE
Commands with leading whitespace should be accepted

### ISSUE THREAD

**Issue by jorwex** [blinksh/blink#1251]
### Checklist

- [x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration

Blink v13.5.12.262
iPhone XS

### Describe the bug

I have been using an iPhone and swipe-able keyboard more and more, and occasionally a leading space is inserted before a command. Blink does not appear to tolerate this. Underscore character used below in place of space for emphasis:

```
blink> _config
 config: command not found
blink> 
```

Apologies if this is better characterized as a feature. I logged this as a bug because most modern shells accept commands entered this way instead of returning "Command not found."

Bash example:
```
pi@octopi:~ $ bash --version
GNU bash, version 5.0.3(1)-release (arm-unknown-linux-gnueabihf)
Copyright (C) 2019 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

user@host:~ $  ls
file1 file2 file3

user@host:~ $ ls
file1 file2 file3
```

**Comment by rfkortekaas** [comment]
Same as #1243

**Comment by jorwex** [comment]
> Same as #1243

Yikes -- This sure is a duplicate. Apologies for not seeing that (extremely recent) existing entry. We even used the same example 😄 

Glad it has been addressed in #1243.
